### ISSUE TITLE
Support Taproot/bech32m addresses

### ISSUE THREAD

**Issue by petzsch** [btcpayserver/BTCPayServer.Vault#41]
With the bitcoin 2.0.0 app on the ledger nano s it should be possible to create taproot/bech32m addresses. However either the btcpayserver UI or the BTCPayServer.Vault is currently not supporting it.

Can we get an update for this?

**Comment by NicolasDorier** [comment]
we really depends on hwi doing the job https://github.com/bitcoin-core/HWI I think it will be one or two year process to support taproot for hardware.

**Comment by petzsch** [comment]
you include a HWI version that supports taproot now, but something else still seems missing.

**Comment by NicolasDorier** [comment]
yes, I need to improve NBXplorer to include some additional data in PSBT.

**Comment by NicolasDorier** [comment]
This is now supported on btcpayserver.
